docs/excel/show-hide-add-in.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Show or hide an Office Add-in in a shared runtime
+description: 'Learn how to programmatically hide or show the UI of an add-in while it runs continuously'
+ms.date: 03/02/2020
+localization_priority: Normal
+---
+
+# Show or hide an Office Add-in in a shared runtime (preview)
+
+Office Add-ins can include any of the following parts:
+
+- a task pane
+- a UI-less function file
+- an Excel custom function
+
+Until now, if an add-in had more than one of these parts, then each part ran in its own separate JavaScript runtime, with its own global object and global variables.
+
+It's possible for add-ins with two or more parts to share a common JavaScript runtime. This shared runtime feature enables new preview APIs for hiding the task pane part of an add-in while the add-in remains running and for reopening the task pane later.
+
+> [!INCLUDE [Information about using preview APIs](../includes/excel-shared-runtime-preview-note.md)]
+
+## Configure an add-in to use a shared runtime
+
+To configure the add-in to use a shared runtime, see [Configure your Office Add-in to use a shared runtime](configure-your-add-in-to-use-a-shared-runtime.md).
+
+## Show and hide the task pane
+
+The new APIs are in the `Office.addin` namespace. To show the task pane, your code calls `Office.addin.showAsTaskpane()`. Office will display in a task pane the page that you assigned to the resource ID (`resid`) for the task pane. This is the `resid` that you assigned to the `<SourceLocation>` of the `<Action xsi:type="ShowTaskpane">` in the manifest. (See [Configure your Office Add-in to use a shared runtime](configure-your-add-in-to-use-a-shared-runtime.md).)
```

Comment:
`Office.addin` isn't technically a namespace, right? It's a read-only property, like `context`.